src/pybind/ceph_daemon.py

```diff
@@ -191,7 +191,7 @@ def format_dimless(self, n, width):
         unit = 0
         while len("%s" % (int(n) // (1000**unit))) > width - 1:
             if unit >= len(units) - 1:
-                break;
```

Comment:
Uahhh, why has this not been detected by pylint? ... OK, we do not check that file. Bummer.